ocs_ci/ocs/constants.py

```diff
@@ -790,6 +792,11 @@
 DR_S3_SECRET_NAME_PREFIX = "odr-s3secret"
 DR_WORKLOAD_REPO_BASE_DIR = "ocm-ramen-samples"
 DR_RAMEN_CONFIG_MANAGER_KEY = "ramen_manager_config.yaml"
+DRPOLICY_STATUS = "Validated"
+RDR_REPLICATION_POLICY = "async"
+
+# ACM version constants
+ACM_VERSION = "2.7"
```

Comment:
The name should be something different if we move to 2.8 ACM then thing's might not work as expected

Reply:
> The name should be something different if we move to 2.8 ACM then thing's might not work as expected

Any ideas?

Reply:
Removed, not needed